macros/SpecName.ejs

```diff
@@ -467,10 +467,6 @@ var specList = {
         name : 'Document Object Model (DOM) Level 3 XPath Specification',
         url  : 'https://www.w3.org/TR/DOM-Level-3-XPath/'
     },
-    'DOM4': {
```

Comment:
Updated those 3 remaining pages. Our spam filter was apparently not happy with the frequent edits you've done on Friday. You've been added to a group of trusted writers, so that it shouldn't bother you again. 

Probably a discussion topic for our discourse, but I wonder why we display old specs anyway. Sure, it is interesting to know in which spec a feature was defined initially, but that could also be a comment like this: https://developer.mozilla.org/en-US/docs/Web/API/Document/createDocumentFragment